libs/pando/engine/src/node/transform.ts

```
@@ -143,136 +143,34 @@ export function detach(
+  map: (_: AnyNode<I>, old: AnyNode<I>) => AnyNode<O>
 ): AnyNode<O>[] {
   const cache = new Map<AnyNode<I>, AnyNode<O>>()
 
   function internal(n: AnyNode<I>): AnyNode<O> {
     const old = cache.get(n)
     if (old) return old
 
+    const prev = n
     const x = dedupMapArray(n.x, internal)
     const br = dedupMapArray(n.br, internal)
     if (n.x !== x || n.br !== br) n = { ...n, x, br } as any
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)